### PR TITLE
Fix benchmarks and add check so that they won't break again

### PR DESCRIPTION
**Pull request by NikVolf**

### CODE REVIEW

File: bin/node/runtime/src/lib.rs [hunk 1]
```diff
@@ -49,7 +49,7 @@ use sp_runtime::traits::{
 	self, BlakeTwo256, Block as BlockT, StaticLookup, SaturatedConversion,
 	ConvertInto, OpaqueKeys, NumberFor,
 };
-use sp_version::RuntimeVersion;
+pub use sp_version::RuntimeVersion;
```
Comment (bkchr) [inline bin/node/runtime/src/lib.rs:52]:
```suggestion
use sp_version::RuntimeVersion;
```

Please include `sp-version` into `node-benching` directly.

File: bin/node/testing/src/bench.rs [hunk 2]
```diff
@@ -462,10 +463,10 @@ impl BenchKeyring {
 	}
 
 	/// Sign transaction with keypair from this keyring.
-	pub fn sign(&self, xt: CheckedExtrinsic, version: u32, genesis_hash: [u8; 32]) -> UncheckedExtrinsic {
+	pub fn sign(&self, xt: CheckedExtrinsic, spec_version: u32, tx_version: u32, genesis_hash: [u8; 32]) -> UncheckedExtrinsic {
```
Comment (bkchr) [inline bin/node/testing/src/bench.rs:466]: Why is this function not directly used from `node-testing`? It is already exposed and should not be duplicated for no reason.

File: bin/node/testing/src/bench.rs [hunk 3]
```diff
@@ -297,9 +298,8 @@ impl BenchDb {
 			&self.keyring,
 		);
 
-		let version = client.runtime_version_at(&BlockId::number(0))
-			.expect("There should be runtime version at 0")
-			.spec_version;
+		let RuntimeVersion { spec_version, transaction_version, .. } = client.runtime_version_at(&BlockId::number(0))
```
Comment (bkchr) [inline bin/node/testing/src/bench.rs:301]: You can also prevent the import of `sp-version` by calling `version.spec_version` & `version.transaction_version`.

File: bin/node/testing/src/bench.rs (hunk 2 above)
Comment (NikVolf) [reply bin/node/testing/src/bench.rs:466]: `node-testing` uses different keyring

**Pull request merged**